specification/provisioningservices/resource-manager/Microsoft.Devices/2017-08-21-preview/iotdps.json

```diff
@@ -0,0 +1,1600 @@
+				"consumes": [],
+				"produces": ["application/json"],
+				"parameters": [{
+					"$ref": "#/parameters/subscriptionId"
+				}, {
+					"name": "resourceGroupName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"$ref": "#/parameters/api-version"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/ProvisioningServiceDescriptionListResult"
+						}
+					},
+					"default": {
+						"description": "DefaultErrorResponse",
+						"schema": {
+							"$ref": "#/definitions/ErrorDetails"
+						}
+					}
+				},
+				"deprecated": false,
+				"x-ms-pageable": {
+					"nextLinkName": "nextLink"
+				}
+			}
+		},
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/provisioningServices/{provisioningServiceName}/skus": {
+			"get": {
+				"tags": ["GET"],
+				"operationId": "IotDpsResource_GetValidSkus",
+				"summary": "Get the list of valid SKUs for a provisioning service.",
+				"description": "Get the list of valid SKUs for a provisioning service.",
+				"parameters": [{
+					"name": "provisioningServiceName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"$ref": "#/parameters/subscriptionId"
+				}, {
+					"name": "resourceGroupName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"$ref": "#/parameters/api-version"
+				}],
+				"responses": {
+					"200": {
+						"description": "This is a synchronous operation. The body contains a JSON-serialized array of the valid SKUs for this provisioning service.",
+						"schema": {
+							"$ref": "#/definitions/IotDpsSkuDefinitionListResult"
+						}
+					},
+					"default": {
+						"description": "DefaultErrorResponse",
+						"schema": {
+							"$ref": "#/definitions/ErrorDetails"
+						}
+					}
+				},
+				"deprecated": false,
+				"x-ms-pageable": {
+					"nextLinkName": "nextLink"
+				}
+			}
+		},
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/provisioningServices/{provisioningServiceName}/certificates/{certificateName}/generateVerificationCode": {
+			"post": {
+				"tags": ["POST"],
+				"operationId": "DpsCertificate_GenerateVerificationCode",
+				"consumes": [],
+				"produces": ["application/json"],
+				"parameters": [{
+					"name": "certificateName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"$ref": "#/parameters/subscriptionId"
+				}, {
+					"name": "resourceGroupName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "provisioningServiceName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "certificate.name",
+					"in": "query",
+					"required": false,
+					"type": "string"
+				}, {
+					"name": "certificate.rawBytes",
```

Comment:
why are all these (rawbytes, isVeirfied etc.) sent as a query parameter instead of in the request body?